install_config/imagestreams_templates.adoc

```diff
@@ -0,0 +1,318 @@
+make it easy for developers to create new applications. By default, the
+link:../install_config/install/quick_install.html[quick installation] and
+link:../install_config/install/advanced_install.html[advanced installation]
+methods automatically create these sets in the *openshift* project, which is a
+default global project to which all users have view access.
+endif::[]
+
+ifdef::openshift-origin[]
+You can populate your OpenShift installation with a useful set of
+Red Hat-provided
+link:../architecture/core_concepts/builds_and_image_streams.html#image-streams[image streams] and
+link:../architecture/core_concepts/templates.html[templates] to
+make it easy for developers to create new applications. By default, the
+link:../install_config/install/advanced_install.html[advanced installation] method automatically creates
+these sets in the *openshift* project, which is a default project to which all
+users have view access.
+endif::[]
+
+ifdef::openshift-enterprise[]
+The core set of image streams and templates are provided and supported by
+Red Hat with an active OpenShift Enterprise subscription for the following
+technologies:
+endif::[]
+ifdef::openshift-origin[]
+Image streams and templates are provided for the following technologies:
+endif::[]
+
+[horizontal]
+Languages::
+- link:../using_images/s2i_images/nodejs.html[Node.js]
+- link:../using_images/s2i_images/perl.html[Perl]
+- link:../using_images/s2i_images/php.html[PHP]
+- link:../using_images/s2i_images/python.html[Python]
+- link:../using_images/s2i_images/ruby.html[Ruby]
+Database::
+- link:../using_images/db_images/mongodb.html[MongoDB]
+- link:../using_images/db_images/mysql.html[MySQL]
+- link:../using_images/db_images/postgresql.html[PostgreSQL]
+Other Services::
+- link:../using_images/other_images/jenkins.html[Jenkins]
+
+ifdef::openshift-enterprise[]
+If you also have the relevant xPaaS Middleware subscription active on your
+account, image streams and templates are also provided and supported by Red Hat
+for each of following middleware services:
+
+[horizontal]
+Middleware Services::
+- link:../using_images/xpaas_images/eap.html[JBoss EAP]
+- link:../using_images/xpaas_images/a_mq.html[JBoss A-MQ]
+- link:../using_images/xpaas_images/jws.html[JBoss Web Server]
+- link:../using_images/xpaas_images/fuse.html[JBoss Fuse Integration Services]
+- link:../using_images/xpaas_images/decision_server.html[Decision Server]
+- link:../using_images/xpaas_images/data_grid.html[JBoss Data Grid]
+endif::[]
+
+You can check which default image streams and templates are currently available
+in your OpenShift environment by logging into the web console and clicking *Add
+to Project*, or getting the lists for the *openshift* project using the CLI:
+
+----
+$ oc get is -n openshift
+$ oc get templates -n openshift
+----
+
+The JSON object definitions used to create these sets are installed on the file
+system of your master during
```

Comment:
It's both if installed using the Quick or Advanced install. For OSE we know they must have used one of those methods, for Origin they can use Advanced (not Quick), but also may have installed using one of the 3 methods in https://docs.openshift.org/latest/getting_started/administrators.html, which would not lay down any example files.